### PR TITLE
UI for questions

### PR DESCRIPTION
**Pull request by shahmharsh**
- Remove AddQuestionActivity
- Added intDef for question options
- Changed CreateMemoryAdapter to allow adding questions
- Pending work
	- User cannot enter options
	- Saving any edits done after the adding a question
	- Saving all questions to the person

@raincoatforever

### CODE REVIEW

**Comment by shahmharsh** [comment]
screenshot_1489280199
screenshot_1489280227

File: app/src/main/java/co/minesweepers/birthday/model/Person.java [hunk 1]
```diff
@@ -54,6 +54,14 @@ public void addQuestion(@NonNull Question question) {
         mQuestions.add(question);
     }
 
+    public void addQuestion(int position, @NonNull Question question) {
```
Comment (raincoatforever) [inline app/src/main/java/co/minesweepers/birthday/model/Person.java]: change to push

File: app/src/main/java/co/minesweepers/birthday/adapters/CreateMemoryAdapter.java [hunk 2]
```diff
@@ -1,121 +1,341 @@
...
+
+            mEditTextQuestion = (EditText) view.findViewById(R.id.edit_text_question);
+            mEditTextQuestion.setOnFocusChangeListener(this);
+
+            RadioButton radioButtonOption1 = (RadioButton) view.findViewById(R.id.radio_option1);
+            RadioButton radioButtonOption2 = (RadioButton) view.findViewById(R.id.radio_option2);
+            RadioButton radioButtonOption3 = (RadioButton) view.findViewById(R.id.radio_option3);
+            RadioButton radioButtonOption4 = (RadioButton) view.findViewById(R.id.radio_option4);
+            mRadioButtonsList = new ArrayList<>(4);
+            mRadioButtonsList.add(radioButtonOption1);
```
Comment (raincoatforever) [inline app/src/main/java/co/minesweepers/birthday/adapters/CreateMemoryAdapter.java]: mRadioButtonsList..addAll(Arrays.asList(radioButtonOption1, radioButtonOption2, radioButtonOption3, radioButtonOption4))

File: app/src/main/java/co/minesweepers/birthday/adapters/CreateMemoryAdapter.java [hunk 3]
```diff
@@ -1,121 +1,341 @@
...
+            }
+            setCheckedRadioButton(0);
+        }
+
+        Question getQuestion() {
+            if (mQuestion != null) {
+                return mQuestion;
+            }
+
+            Question.Builder builder = new Question.Builder()
```
Comment (raincoatforever) [inline app/src/main/java/co/minesweepers/birthday/adapters/CreateMemoryAdapter.java:222]: could directly build from question . 

new Question.Builder().buildFrom(Question question). makes intention clear

**Comment by raincoatforever** [comment]
👍

**Pull request merged**